opennms-base-assembly/src/main/filtered/bin/install

```diff
@@ -6,12 +6,18 @@ OPENNMS_BINDIR="${install.bin.dir}"
 RUNAS="root"
 
 ULIMIT=`which ulimit 2>/dev/null`
```

Comment:
Good catch.  I just changed it to execute `ulimit` instead and then check the return value.